libraries/SITL/examples/Webots_Python/params/heron.parm

```diff
@@ -0,0 +1,13 @@
+FRAME_CLASS      2          # Boat
+
+SERVO1_FUNCTION  73         # Throttle Left (Boat throttle channel)
+SERVO2_FUNCTION  74         # Throttle Right (Boat throttle channel)
+SERVO3_FUNCTION  0          # Disabled (no steering required)
+SERVO4_FUNCTION  0          # Disabled (no steering required)
+
+AHRS_EKF_TYPE    10         # Use sim AHRS
+#ARMING_CHECK	 1045534    # No RC requirements
```

Comment:
Why remove RC here?  Simulation has it by default?

If this was copied from elsewhere that's fine.